pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -119,12 +119,13 @@ func checkIfUserPartOfHyperVAdmins() error {
 }
 
 func fixUserPartOfHyperVAdmins() error {
-	outGroupName, _, err := powershell.Execute(`(New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")).Translate([System.Security.Principal.NTAccount]).Value`)
+	_, _, err := powershell.ExecuteAsAdmin("adding current user to Hyper-V administrator group", fmt.Sprintf("Add-LocalGroupMember -SID 'S-1-5-32-578' -Member '%s'", username()))
```

Comment:
Since we were executing this command without any admin privileges does this show  popup to user  now?

Reply:
yes, UAC pop-up is shown, i think it'll be better to club the two "adding to crc-users" as well as "adding to Hyper-V admins" group would be better, we'll get only one UAC pop-up then

Reply:
But then does it also catch individual action error? iirc if we club 2-3 action in same then error catching is difficult.

Reply:
No we can have combined stdout o/p from both the commands, but it'll not be able catch error from individual actions. i think in this case its an acceptable trade-off, the two actions are doing similar thing of adding current member to a group, so its easy to narrow down the issue